src/plugins/vis_default_editor/tsconfig.json

```diff
@@ -12,7 +12,8 @@
   "references": [
     { "path": "../../core/tsconfig.json" },
     { "path": "../data/tsconfig.json" },
-    { "path": "../visualize/tsconfig.json" },
+    { "path": "../visualizations/tsconfig.json" },
+    { "path": "../discover/tsconfig.json" },
```

Comment:
I think in case of adding `visualizations/tsconfig.json` it should still work without `/discover/tsconfig.json`
but for that case I prefer to keep discover cause it's in `requiredBundles`.

![image](https://user-images.githubusercontent.com/20072247/148219853-cf4b2133-c504-45d0-ad65-d5a282f179c3.png)